ui-ngx/src/app/modules/home/components/widget/lib/multiple-input-widget.component.ts

```diff
@@ -597,4 +603,25 @@ export class MultipleInputWidgetComponent extends PageComponent implements OnIni
     });
     this.multipleInputFormGroup.markAsPristine();
   }
+
+  openEditJSONDialog($event: Event, formKey: string) {
+    if ($event) {
+      $event.stopPropagation();
+    }
+    const formControl = this.multipleInputFormGroup.controls[formKey];
+    this.dialog.open<JsonObjectEditDialogComponent, JsonObjectEditDialogData, object>(JsonObjectEditDialogComponent, {
+      disableClose: true,
+      panelClass: ['tb-dialog', 'tb-fullscreen-dialog'],
+      data: {
+        jsonValue: formControl.value
+      }
```

Comment:
Dialog does not support settings buttons labels for 'SAVE' and 'CANCEL' edits. Need to add settings to set title dialog